packages/supi/src/install/index.ts

```diff
@@ -837,8 +837,16 @@ async function installInContext (
       })
     }
 
-    if (!opts.lockfileOnly) {
-      await Promise.all(projectsToLink.map(linkBinsOfImporter))
+    const binNames = (await Promise.all(projectsToLink.map(linkBinsOfImporter))).flat()
```

Comment:
why have you removed the lockfileOnly if statement?

Reply:
because this statement is already inside the same condition on L811